packages/create-freesewing-pattern/template/freesewing/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```

Comment:
I'm actually not positive about this one, but feel like it's likely not intended to be stored in git.

Reply:
You feel right :)